tests/nvme_test.py

```diff
@@ -36,6 +36,18 @@
 from nvme_test_logger import TestNVMeLogger
 
 
+def hex_to_decimal(value):
+    """ Wrapper for converting hexadecimal numbers to decimal
+        - Args:
+            - value: A number either in hex or base 10
+        - Returns:
+            - Decimal integer
+    """
+    if str(value).startswith("0x"):
```

Comment:
You could set the base to 0 to do the same thing as checking if the string starts with 0x.
```
>>> print(f"{int('10', 0)}")
10
>>> print(f"{int('0x10', 0)}")
16
```

Reply:
Thanks, this is very neat!
Changed it :)